.github/workflows/deploy-and-migrate.yml

```
@@ -0,0 +1,48 @@
+          check-interval: 10
+          max-wait: 300
+          environment: Production
+
+  migrate:
+    name: Push Drizzle Schema to DB
+    needs: wait-for-vercel-deploy
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)